Python/specialize.c

```diff
@@ -792,6 +824,23 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name)
         case GETSET_OVERRIDDEN:
             SPECIALIZATION_FAIL(LOAD_ATTR, SPEC_FAIL_OVERRIDDEN);
             goto fail;
+        case GET_CALL_GETATTRIBUTE:
+        {
+            assert(type->tp_getattro == _Py_slot_tp_getattro);
+            assert(Py_IS_TYPE(descr, &PyFunction_Type));
+            _PyLoadMethodCache *lm_cache = (_PyLoadMethodCache *)(instr + 1);
+            uint32_t func_version = function_get_version(descr, 2,
```

Comment:
See comment in `function_get_version`, but this does two things.
``
uint32_t func_version = `function_check_args(descr, 2) && function_get_version(descr)`

Reply:
This is really smart! Thanks.